yarn-project/p2p/Dockerfile

```diff
@@ -1,17 +1,19 @@
 FROM 278380418400.dkr.ecr.eu-west-2.amazonaws.com/yarn-project-base AS builder
 
-COPY p2p p2p
-# dependencies
-COPY circuits.js circuits.js
-COPY ethereum.js ethereum.js
-COPY foundation foundation
-COPY l1-contracts l1-contracts
-COPY l2-block l2-block
-COPY tx tx
-COPY unverified-data unverified-data
+COPY . .
```

Comment:
+1 if we're going to have a dependency system no reason to copy dependencies one by one (which mostly can help with docker figuring out its own dependencies)

Reply:
https://github.com/AztecProtocol/aztec3-packages/issues/224